core-bundle/src/EventListener/SwitchUserListener.php

```diff
@@ -59,9 +60,26 @@ public function __invoke(SwitchUserEvent $event): void
             $targetUser = $targetUser->getUsername();
```

Comment:
`$targetUser` is always `instanceof UserInterface`, not sure why this check is here.

Reply:
Yes, this seems to be right. I will remove the check.